examples/web/nextjs-chat/src/app/api/momento/token/config.ts

```diff
@@ -17,7 +17,7 @@ import {
  * 
  * TokenScopes provides several functions that will return the permissions you
  * request for a given cache and topic name.
- *    export const tokenPermissions: TokenScope = TokenScopes.topicPublishSubscribe("default-cache", AllTopics);
+ *    export const tokenPermissions: TokenScope = TokenScopes.topicPublishSubscribe(AllCaches | "your-cache-name", AllTopics);
```

Comment:
in case folks are tempted to copy/paste this, I wouldn't use the `|` here because it will do something surprising :)
I would just have two copies of this line, one with `AllCaches` and one with the string version.